independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/UpdateCommandHandler.java

```diff
@@ -0,0 +1,373 @@
+                }
+            }
+            if (importsToBeRemoved) {
+                for (PlatformInfo importInfo : platformImports.values()) {
+                    if (importInfo.recommended == null) {
+                        log.info(String.format(UpdateCommandHandler.PLATFORM_RECTIFY_FORMAT,
+                                UpdateCommandHandler.REMOVE, importInfo.imported.toCompactCoords()));
+                    }
+                }
+            }
+            log.info("");
+        }
+
+        final Map<ArtifactKey, ExtensionInfo> extensionInfo = new LinkedHashMap<>(currentState.getExtensions().size());
+        for (TopExtensionDependency dep : currentState.getExtensions()) {
+            extensionInfo.put(dep.getKey(), new ExtensionInfo(dep));
+        }
+        for (TopExtensionDependency dep : recommendedState.getExtensions()) {
+            final ExtensionInfo info = extensionInfo.get(dep.getKey());
+            if (info != null) {
+                info.recommendedDep = dep;
+            }
+        }
+        final Map<ArtifactCoords, List<ArtifactCoords>> versionedManagedExtensions = new LinkedHashMap<>(0);
+        final Map<String, List<ExtensionInfo>> nonPlatformExtensionUpdates = new LinkedHashMap<>();
+        for (ExtensionInfo info : extensionInfo.values()) {
+            if (info.getRecommendedDependency() == null) {
+                continue;
+            }
+            if (info.getRecommendedDependency().isPlatformExtension()) {
+                if (info.currentDep.isNonRecommendedVersion()) {
+                    versionedManagedExtensions
+                            .computeIfAbsent(info.getRecommendedDependency().getOrigin().getBom(), k -> new ArrayList<>())
+                            .add(info.currentDep.getArtifact());
+                }
+            } else if (!info.currentDep.getVersion().equals(info.getRecommendedDependency().getVersion())) {
+                nonPlatformExtensionUpdates.computeIfAbsent(info.currentDep.getProviderKey(), k -> new ArrayList<>()).add(info);
+            }
+        }
+
+        if (!versionedManagedExtensions.isEmpty()) {
+            for (Map.Entry<ArtifactCoords, List<ArtifactCoords>> origin : versionedManagedExtensions.entrySet()) {
+                log.info("Extensions from " + origin.getKey().toCompactCoords() + ":");
+                for (ArtifactCoords c : origin.getValue()) {
+                    final StringBuilder sb = new StringBuilder();
+                    sb.append(String.format(UpdateCommandHandler.PLATFORM_RECTIFY_FORMAT,
+                            UpdateCommandHandler.UPDATE, c.toCompactCoords()));
+                    sb.append(" -> ");
+                    sb.append(c.getGroupId()).append(':')
+                            .append(c.getArtifactId());
+                    if (!c.getClassifier().isEmpty()) {
+                        sb.append(':').append(c.getClassifier());
+                    }
+                    log.info(sb.toString());
+                }
```

Comment:
I think it was this one. If the recommendation is to remove the version (shown in the original snippet for the item), the display: GAV -> GA .. doesn't explain to me what I should do (which is remove the version specifier).

Reply:
I've changed to ` -> remove version (managed)` for now as discussed.